sonobuoy-conformance-1.7.yaml

```diff
@@ -0,0 +1,194 @@
+---
```

Comment:
were updating https://scanner.heptio.com/ for 1.8 right now.  Should have an update in about a week.

Reply:
Cool.  Happy to update the instructions to refer to scanner whenever.  For now, I'm keeping it simple with this static yaml file.